HotTips/TipOfTheDayWindow.xaml.cs

```diff
@@ -198,7 +199,11 @@ internal bool NavigateToTip(TipInfo nextTip, bool markAsSeen = true)
                 return false;
             }
 
-            currentTip = nextTip.globalTipId;
+            TipHistoryInfo tipHistoryObj = new TipHistoryInfo();
+            tipHistoryObj.globalTipId = nextTip.globalTipId;
+            tipHistoryObj.tipLikeStatus = TipLikeEnum.NORMAL;
+
+            currentTip = tipHistoryObj;
 
             // Render the new tip content in the tip viewer
             ShowTipContent(nextTip);
```

Comment:
it would be useful if nextTip was a TipHistoryInfo item, because then you would not need to look up the history later for the voting.

Reply:
Here the nextTip is a TipInfo. It contains contentUri but TipHistoryInfo does not. Hence, I left as it is.